test/cpp/lazy/test_lazy_ops.cpp

```diff
@@ -19,7 +19,12 @@ namespace lazy {
 
 namespace {
   // This registers the torchscript backend, without which lazy device won't work
-  torch::lazy::BackendRegistrar g_registrar(GetTSBackendImpl());
+static bool inline init_backend(){
+  torch::lazy::InitTorchScriptBackend();
+  return true;
+}
+static const bool backend_initialized = init_backend();
```

Comment:
It looks like ASAN isn't happy (the ASAN test is complaining about initialization-order-fiasco: https://github.com/pytorch/pytorch/runs/5746632034?check_suite_focus=true)

I think the ordering problem might be between this static bool (which eventually calls `RegisterTorchScriptLazyNativeFunctions`), and the static library object that that function uses in the codegen: `static auto m = MAKE_TORCH_LIBRARY_IMPL(aten, $dispatch_key);`.

You might need to either access `m` through a function, or maybe not make this bool static?